src/components/navigation/_macro.njk

```diff
@@ -88,7 +88,7 @@
             <nav class="ons-navigation ons-navigation--sub-mobile ons-u-d-no ons-js-secondary-nav ons-u-mt-xs" id="{{ params.navigation.subNavigation.id }}--mobile" aria-hidden="true" aria-label="{{ params.navigation.subNavigation.ariaLabel | default("Section menu") }}" data-analytics="header-section-navigation">
                 <div class="ons-container ons-container--gutterless@xxs@l{{ ' ons-container--full-width' if params.navigation.fullWidth }}{{ ' ons-container--wide' if params.navigation.wide }}">
                     <ul class="ons-navigation__list ons-navigation__list--parent">
-                        <li class="ons-navigation__item">
+                        <li class="ons-navigation__item{{ ' ons-navigation__item--active' if (params.navigation.subNavigation.overviewURL == params.navigation.subNavigation.currentPath)}}">
```

Comment:
```suggestion
                        <li class="ons-navigation__item{{ ' ons-navigation__item--active' if (params.navigation.subNavigation.overviewURL == params.navigation.subNavigation.currentPath) or (params.navigation.subNavigation.overviewURL == params.navigation.currentPath) or (params.navigation.currentPath is not string and params.navigation.subNavigation.overviewURL in params.navigation.currentPath) }}">
```
This change isn't working for me. But I updated it to this and after testing it, it works.